### ISSUE TITLE
A Native Collection has not been disposed

### ISSUE THREAD

**Issue by tichise** [google-ar/arcore-unity-extensions#79]
I am encountering a bug where native collections are not being disposed of. I am using the version ARCore Extensions for AR Foundation v1.32.0.

```
A Native Collection has not been disposed, resulting in a memory leak. Allocated from:
Unity.Collections.NativeArray`1:.ctor(Byte[], Allocator) (at /Users/bokken/buildslave/unity/build/Runtime/Export/NativeArray/NativeArray.cs:69)
UnityEngine.Networking.UploadHandlerRaw:.ctor(Byte[]) (at /Users/bokken/buildslave/unity/build/Modules/UnityWebRequest/Public/UploadHandler/UploadHandler.bindings.cs:95)
Google.XR.ARCoreExtensions.Editor.Internal.ARCoreAnalytics:SendAnalytics(String, LogRequest) (at Library/PackageCache/com.google.ar.core.arfoundation.extensions@abb3767358/Editor/Scripts/Internal/ARCoreAnalytics.cs:105)
Google.XR.ARCoreExtensions.Editor.Internal.ARCoreAnalytics:.cctor() (at Library/PackageCache/com.google.ar.core.arfoundation.extensions@abb3767358/Editor/Scripts/Internal/ARCoreAnalytics.cs:55)
```

**Comment by devbridie** [comment]
Resolved in [1.36.0](https://github.com/google-ar/arcore-unity-extensions/releases/tag/v1.36.0) by https://github.com/google-ar/arcore-unity-extensions/pull/102. Thanks!

**Comment by tichise** [comment]
@devbridie Thanks for the response. I have confirmed that the corrected version is out.
